zebra/zebra_script.c

```diff
@@ -329,14 +329,6 @@ void lua_pushzebra_dplane_ctx(lua_State *L, const struct zebra_dplane_ctx *ctx)
 		lua_setfield(L, -2, "ipset");
 		break;
 	}
-	case DPLANE_OP_ADDR_INSTALL:
```

Comment:
why are we moving this around ?

Reply:
So that all the currently unhandled stuff is in one place. It's more readable imo for when someone comes later and says "what doesnt have lua conversion support"

Reply:
let's get this in because the build is broke but let's break commits up like this into 2 separate commits for the future.  the fix and then the code reorg